src/main/groovy/org/labkey/gradle/plugin/Jsp.groovy

```diff
@@ -108,8 +112,8 @@ class Jsp implements Plugin<Project>
         // it for the command line, there will be lots of warnings about .tld files on the classpath where
         // they don't belong ("CLASSPATH element .../labkey.tld is not a JAR.").  These warnings may appear if
         // building within IntelliJ but perhaps we can live with that (for now).
-        if (BuildUtils.isIntellij())
-            project.dependencies.add("implementation", project.rootProject.tasks.copyTagLibsBase.inputs.files)
+//        if (BuildUtils.isIntellij())
+//            project.dependencies.add("implementation", getCopyTagLibsBase(project.gradle).get().inputs.files)
```

Comment:
Can you move the comments to where this code moved to and remove the commented out code?

Reply:
Done